src/Sulu/Bundle/ContactBundle/Resources/config/lists/accounts.xml

```diff
@@ -238,12 +245,52 @@
             <field-name>id</field-name>
             <entity-name>SuluTagBundle:Tag</entity-name>
 
+            <joins ref="tags"/>
+        </group-concat-property>
+
+        <!-- tagId property is only used for filtering and should never be included as column in a list -->
+        <!-- when used as column, the property will lead to duplicated rows for entities with multiple tags -->
```

Comment:
TODO: add a testcase to ensure that this does not happen when only used for filtering

Reply:
this depends on #5943